### PR TITLE
win: add dch 536.23

### PR DESCRIPTION
**Pull request by Umgak**
**Purpose of proposed changes**
Mostly another version bump.
Something is up with autopatch.py - directing it to an exe outputs an offset off by 400 bytes.

**Essential steps taken**

Generated with autopatch.py, tested with patch-tester.ps1. Readme generated with add_driver.py and readme_autogen.py. The basics.

### CODE REVIEW

**Comment by Umgak** [comment]
I recommend that someone who's actually good at Python and isn't working at 1AM look over the autopatch.py again - my fix feels like a bodge. Could implement detection for whether it's a new-format or old-format archive?

File: win/tools/autopatch/autopatch.py [hunk 1]
```diff
@@ -156,7 +156,8 @@ def make_patch(archive, *,
                                  tmpdir,
                                  arch_tgt,
                                  sevenzip=sevenzip) as tgt:
-                f = expand(tgt, sevenzip=sevenzip)
+                 with open(tgt, 'rb') as fo:
+                    f = fo.read()
```
Comment (jailuthra) [inline win/tools/autopatch/autopatch.py:160]:
```suggestion
                if tgt.endswith(".dll"):
                    with open(tgt, 'rb') as fo:
                        f = fo.read()
                else:
                    f = expand(tgt, sevenzip=sevenzip)
```

**Review by jailuthra — CHANGES_REQUESTED** [review]
Thanks @Umgak for figuring this out.

Your changes LGTM, but I'm concerned if someone has to extract or patch an older driver they might not be able to just pass '-T something.dl_` anymore. Maybe we can instead support both modes by adding an if/else check based on the extension of the target?

**Review by jailuthra — APPROVED** [review]

**Comment by reloxx13** [comment]
Thank you very much! Tested today and works 🎉

**Pull request merged**